docs/research/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Research
+---
+
+In the following you will find all information about Saros in the context of research.<br/>
+**If you are interested to use Saros in the course of your own research, feel free to [contact us](../support).**
+
+## Research topics and questions
+The [AG SE (working group software engineering *@FU Berlin*)](http://www.inf.fu-berlin.de/w/SE/WebHome) has been doing research on pair programming for a number of years. Saros is a platform for extending this work to distributed pair programming (and general collaborative programming). We also investigate a number of research questions that concern the Saros tool itself (and similar tools).
+ 
+### How to introduce distributed pair programming? Effects? 
+We are working with companies that perform distributed software engineering and are interested in collaborative programming (via Saros) as a means for abating some of the problems that this tends to bring along.
+We are also studying how to introduce Saros and DPP into Open-Source projects and what effects the use of Saros has on such projects.
+ 
+### How should Saros work? 
+Saros has a number of interesting usability aspects in a number of areas, for instance:
+* learning its concepts and base operation
+* useful functionality beyond mere text editing
+* practical forms of session management
+* awareness of what the other participants are doing We are studying these aspects via multiple channels such as gathering written user feedback, collecting and analyzing fine-grained usage data, and explicit usability studies.
+ 
+### Saros usage statistics and diagrams 
+Saros includes a statistical framework which is responsible for gathering information about the actual usage. This information ranges from the plain duration of a session to much more detailed facets such as role swaps performed or text edits contributed. For each session - assuming one allows the submission of this anonymous information - a session statistic is sent to our server.
```

Comment:
Maybe mention that this is not currently in use? Even though the functionality theoretically exists, the infrastructure to accommodate it is not currently set up if I remember correctly.